Changes the Squid default proxy port to a realm-specific value.

Index: squid/squid.conf
===================================================================
--- squid.orig/squid.conf
+++ squid/squid.conf
@@ -5693,7 +5693,7 @@ http_access deny all
 
 always_direct allow from_localhost to_localhost
 
-http_port 3128
+http_port localhost:8080
 
 hierarchy_stoplist cgi-bin ?
 coredump_dir /var/cache/squid
